byzanz-gui

```diff
@@ -17,16 +18,11 @@ TIME=$(date +"%Y-%m-%d_%H%M%S")
 DELAY=10
  
 # Standard screencast folder
-FOLDER="$HOME/Pictures"
+FOLDER=$(xdg-user-dir PICTURES 2>/dev/null || echo "$HOME/Pictures")/ByzanzGif
```

Comment:
Was it a problem to drop the files into ~/Pictures? I'd prefer to keep ~/Pictures or have it configurable. We could make all of these "constants" read from the environment and use user-provided input if that's something useful.

Example:

```
FOO="${FOO-default}"

FOLDER="${FOLDER-$(xdg-user-dir PICTURES 2>/dev/null || echo "$HOME/Pictures")}"
```

(note: I didn't test the above, there might be a mistake or two ;) )